api/AudioWorklet.json

```diff
@@ -17,10 +17,10 @@
             "version_added": null
           },
           "firefox": {
-            "version_added": null
+            "version_added": true
```

Comment:
Errors of the same type as in https://github.com/mdn/browser-compat-data/pull/3658 seemed quite likely here, and this is the first I could spot. `AudioWorklet` may be in Gecko's .webidl files, but it's not shipped in Firefox 65 or in 66 beta.

Reply:
Actually, upon further inspection, `AudioWorklet` was introduced in Firefox 63 (though `AudioWorkletGlobalScope` has an `[Exposed=AudioWorklet]` back in 54 even though `AudioWorklet` itself is not defined).  See [bug 1470856](https://bugzilla.mozilla.org/show_bug.cgi?id=1470856) and the corresponding [diff in Mercurial](https://hg.mozilla.org/mozilla-central/rev/74e0bf980eae).

Reply:
The diff also contains two prefs "dom.audioworklet.enabled" and "dom.worklet.enabled". So this implemented but likely only behind these prefs.